### PR TITLE
Figure Size Control

### PR DESCRIPTION
**Pull request by blink1073**
Adds figure size control and a `-S` or `--size` option for `%matlab` magic.

### CODE REVIEW

**Comment by arokem** [comment]
This one needs a rebase to be merged

**Comment by blink1073** [comment]
All set!

**Comment by blink1073** [comment]
It occurs to me that I should rerun the example notebooks and show `--size` in action, agreed?

File: pymatbridge/matlab_magic.py [hunk 1]
```diff
@@ -102,6 +102,7 @@ def __init__(self, shell,
 
         self.Matlab = pymat.Matlab(matlab, maxtime=maxtime)
         self.Matlab.start()
+        self._initialized = False
```
Comment (arokem) [inline pymatbridge/matlab_magic.py]: I am not sure why this is here. Isn't this an attribute of `self.Matlab`?

File: pymatbridge/pymatbridge.py [hunk 2]
```diff
@@ -169,6 +169,7 @@ def __init__(self, executable, socket_addr=None,
 
         self.context = None
         self.socket = None
+        self._initialized = False
```
Comment (arokem) [inline pymatbridge/pymatbridge.py]: And not sure what role this plays, either. Wouldn't you rather just call `set_default_plot_size` in `start`?

**Comment by blink1073** [comment]
Cleaned up, and reran the notebooks.

**Comment by arokem** [comment]
Awesome.

**Pull request merged**